### PR TITLE
Python gettext localize master

### PR DESCRIPTION
**Pull request by c-holtermann**
Patches of PR #655 applied on master.

### CODE REVIEW

**Comment by c-holtermann** [comment]
This is still the same state as #655 - needs work.

**Comment by c-holtermann** [comment]
@jralls would you please try this version?

File: bindings/python/tests/CMakeLists.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ endif()
 
 set(test_python_bindings_DATA
         runTests.py.in
+	test_gettext.py
```
Comment (gjanssens) [inline bindings/python/tests/CMakeLists.txt:19]: Cosmetic: can you use spaces instead of a tab here ? That will align the file with the others.

**Comment by jralls** [comment]
It works. Please clean out the tabs (some programming editors make that easy, e.g. emacs has `M-X untabbify`, vim has `:retab`) and force push and I'll merge it. (Note that feature freeze for 4.0 is this weekend so it would be *really good* to get it in by then.)

**Comment by c-holtermann** [comment]
> 
> 
> Looks good. Do you intend to work on the unit test still ?
> Also, you may have to rerun make or ninja and commit an updated POTFILES.in afterwards. If I run ninja right after merging your branch a few lines change in that file.

I don't have gettext 0.2. I'd be glad if someone could supply me with POTFILES.in after this PR has been merged.

File: bindings/python/tests/CMakeLists.txt (hunk 1 above)
Comment (c-holtermann) [reply bindings/python/tests/CMakeLists.txt]: Sure. I'm using vim and on python files that gets done automatically. But this is a txt.

**Comment by jralls** [comment]
Thanks!

**Comment by fellen** [comment]
> I don't have gettext 0.2. I'd be glad if someone could supply me with POTFILES.in after this PR has been merged.

Sent by private email.

**Pull request merged**